README.md

```diff
@@ -31,15 +32,17 @@ name: PR lint
         github-token: ${{ secrets.GITHUB_ACCESS_TOKEN }}
         pivotal-token: ${{ secrets.PIVOTAL_TOKEN }}
         skip-branches: '^(production-release|master|release\/v\d+)$'
+        skip-comments: true
```

Comment:
maybe `comment-large-pr: true` and `comment-pr-title: true`? I think more people are likely to use the `comment-large-pr: true` setting..

Later we can use a number here for a threshold..

Reply:
sure. ill do this in the next release. we should provide an option to set the limit